src/components/pages/ProjectInfo/ProjectInfo.js

```diff
@@ -38,140 +33,52 @@ import { useHttpResponseErrorHandler } from '../../../App/HttpResponseErrorHandl
-`
-const ClearTagButton = styled(CloseButton)`
-  position: relative;
-  color: ${theme.color.textColor};
-  opacity: 0;
-  transition: 0;
-  &:focus {
-    opacity: 1;
-  }
-  &:hover,
-  &:focus {
-    & + span {
-      display: block;
-    }
-  }
-`
-const TagStyle = styled.li`
-  position: relative;
-  color: ${theme.color.textColor};
-  border-radius: 50px;
-  background-color: ${theme.color.white};
-  padding: 0 4rem 0 0;
-  margin: 1rem 0.5rem;
-  border: solid ${theme.spacing.borderMedium} ${theme.color.primaryColor};
-  display: inline-block;
-  white-space: nowrap;
-  &:focus {
-    ${ClearTagButton} {
-      opacity: 1;
-    }
-  }
-  ${hoverState(css`
-    ${ClearTagButton} {
-      opacity: 1;
-    }
-  `)}
-  @media (hover: none) {
-    ${ClearTagButton} {
-      opacity: 1;
-    }
-  }
-`
-
-const TooltipPopup = styled('span')`
-  display: none;
-  background: ${theme.color.primaryColor};
-  color: ${theme.color.white};
-  position: absolute;
-  font-size: ${theme.typography.smallFontSize};
-  clip-path: polygon(
-    calc(20px - 10px) 15px,
-    20px 0,
-    calc(20px + 10px) 15px,
-    100% 15px,
-    100% 100%,
-    0 100%,
-    0 15px
-  );
-  padding: ${theme.spacing.small};
-  padding-top: calc(4rem - 15px);
-  top: 4rem;
-  white-space: normal;
-  text-align: start;
-  line-height: ${theme.typography.lineHeight};
-  z-index: 101;
-  ${theme.typography.upperCase};
-`
-const InputAutocompleteWrapper = styled(InputRow)`
-  height: 100px;
-`
-
-const OrganizationList = ({ organizations, handleOrganizationsChange }) => {
-  return (
-    organizations && (
-      <TagStyleWrapper>
-        {organizations.map((item) => {
-          const uid = createUuid()
-
-          return (
-            <TagStyle tabIndex="0" key={item}>
-              <ClearTagButton
-                type="button"
-                onClick={() => handleOrganizationsChange(item)}
-                id={`remove-button-${uid}`}
-                aria-labelledby={`aria-tooltip-label${uid}`}
-              >
-                <IconClose />
-              </ClearTagButton>
-              <TooltipPopup id={`aria-tooltip-label${uid}`}>
-                {language.pages.projectInfo.removeOrganization}
-              </TooltipPopup>
-              {item}
-            </TagStyle>
-          )
-        })}
-      </TagStyleWrapper>
-    )
-  )
-}
+import { EditCitationModal } from './EditCitationModal'
+import {
+  BlockquoteInForm,
+  InputAutocompleteWrapper,
+  SuggestNewOrganizationButton,
+} from './ProjectInfo.styles'
+import { OrganizationList } from './OrganizationsList'
+
+const getWhichServerCitationToUse = (project) =>
+  project?.user_citation // false if empty string, which is how the server stores undefined
+    ? project?.user_citation
+    : project?.default_citation
 
 const ProjectInfo = () => {
   const [idsNotAssociatedWithData, setIdsNotAssociatedWithData] = useState([])
+  const [isDeleteProjectModalOpen, setIsDeleteProjectModalOpen] = useState(false)
```

Comment:
Im trying to spare reviewers by not sorting/tidying state and other vars at the top of react components which can often obscure the actual important changes in the diff, but I felt I needed to here.